contracts/identity/Identity.sol

```diff
@@ -171,6 +171,11 @@ contract Identity is ProxyStorage {
         return hash;
     }
 
+    function useHash(bytes32 hash) public {
```

Comment:
I don't like it because I prefer when functions called by other functions are below one another and if I move that with non view functions then `validateNonce()` is no longer right after `executeTransaction()`. But I did it anyway